cstore_reader.c

```diff
@@ -789,7 +789,11 @@ SelectedBlockMask(StripeSkipList *stripeSkipList, List *projectedColumnList,
 							 blockSkipNode->maximumValue);
 
 			constraintList = list_make1(baseConstraint);
+#if (PG_VERSION_NUM >= 100000)
```

Comment:
Last time I could not get it to work. It looks like there has been some changes in tools project. Trying again.